deepchecks/vision/utils/detection_formatters.py

```diff
@@ -28,29 +28,34 @@ class DetectionLabelFormatter(BaseLabelFormatter):
 
     Parameters
     ----------
-    label_formatter : Union[str, Callable]
-        Function or string that specifies the formatting function.
-        If a string, it must be one of the following:
-            - 'xyxy' - x1, y1, x2, y2 represent the upper left and lower right corners of the bounding box.
-            - 'xywh' - x1, y1, w, h represent the upper left corner of the bounding box and its width and height.
-            - 'cxcywh' - cx, cy, w, h represent the center of the bounding box and its width and height.
-            - 'xyxyn' - x1, y1, x2, y2, n represent the upper left and lower right corners of the bounding box,
-                        normalized to the image dimensions.
-            - 'xywhn' - x1, y1, w, h, represent the upper left corner of the bounding box and its width and height,
-                        normalized by the image dimensions.
-            - 'cxcywhn' - (YOLO format) x, y, w, h, represent the center of the bounding box and its width and height,
-                          normalized by the image dimensions.
-        In addition, the label shape should be a list of length N containing tensors of shape (M, 5), where N is the
-        number of samples, M is the number of bounding boxes, and each bounding box is represented by 5 values:
-        (class_id, 4 coordinates in the format specified by the `label_formatter` parameter).
-
-        If a function, it must follow the signature:
+    label_formatter : Callable
         Function that takes in a batch of labels and returns the encoded labels in the following format:
         List of length N containing tensors of shape (M, 5), where N is the number of samples,
```

Comment:
Do you think it would be helpful to add here a link to an external resource explaining the format? (e.g. to COCO's explanation...)